### PR TITLE
Feature/fix swagger url

### PR DESCRIPTION
**Pull request by manuwei**
#### Short Description

  * swagger renaming must consistent
  * fix unspecific error message
  * backendproperties: should be required, link is not required for implementations

### CODE REVIEW

File: org.planqk.atlas.web/src/main/java/org/planqk/atlas/web/dtos/ImplementationDto.java [hunk 1]
```diff
@@ -49,7 +49,6 @@
 
     @NotNull(message = "Implementation-Name must not be null!")
     private String name;
-    @NotNull(message = "Implementation-FileLocation must not be null!")
```
Comment (salmma) [inline org.planqk.atlas.web/src/main/java/org/planqk/atlas/web/dtos/ImplementationDto.java:52]: maybe the reason why building fails, there are test checking required variables. I have already covered this in my branch

File: org.planqk.atlas.web/src/main/java/org/planqk/atlas/web/dtos/BackendDto.java [hunk 2]
```diff
@@ -25,5 +26,6 @@
     private String vendor;
     private String technology;
     private QuantumComputationModel quantumComputationModel;
+    @NotNull()
```
Comment (salmma) [inline org.planqk.atlas.web/src/main/java/org/planqk/atlas/web/dtos/BackendDto.java:29]: already covered this in my branch :)

File: org.planqk.atlas.core/src/main/java/org/planqk/atlas/core/model/BackendPropertyType.java [hunk 3]
```diff
@@ -4,12 +4,16 @@
...
+import lombok.NonNull;
 
+@NoArgsConstructor
 @EqualsAndHashCode(callSuper = true)
 @Data
 @Entity
 public class BackendPropertyType extends HasId {
 
+    @NonNull
     private String name;
```
Comment (salmma) [inline org.planqk.atlas.core/src/main/java/org/planqk/atlas/core/model/BackendPropertyType.java:17]: already covered this in my branch :)

File: org.planqk.atlas.web/src/main/java/org/planqk/atlas/web/dtos/ImplementationDto.java (hunk 1 above)
Comment (manuwei) [reply org.planqk.atlas.web/src/main/java/org/planqk/atlas/web/dtos/ImplementationDto.java:52]: i'll delete it here

File: org.planqk.atlas.web/src/main/java/org/planqk/atlas/web/dtos/BackendDto.java (hunk 2 above)
Comment (manuwei) [reply org.planqk.atlas.web/src/main/java/org/planqk/atlas/web/dtos/BackendDto.java:29]: i'll delete it here

**Pull request merged**